### ISSUE TITLE
Client: Sepolia network connection in v0.3 release not working

### ISSUE THREAD

**Issue by holgerd77** [ethereumjs/ethereumjs-monorepo#1680]
The `Sepolia` network connection is not working when syncing the chain with an installed client `v0.3.0` version, which is particularly annoying, since I relatively prominently pointed this out as a good client testing opportunity in the release announcements.

Here is the output, deleted the `sepolia` folder before and ran `ethereumjs --network=sepolia`:

![grafik](https://user-images.githubusercontent.com/931137/151980024-7621cb80-9821-4ea3-b8a3-ce02f5774ea2.png)

This is actually *not* happening when running the client from within the monorepo.

Someone should give this some prioritization throughout the day so that we can quickly do a bugfix release.

Not sure, one thesis on this: maybe we should have/would need to also to `Block` and `Blockchain` releases, to make sure that these operate on the latest Common as well? (even if these libraries have no other changes included) 🤔

Would need some validation though.

**Comment by ryanio** [comment]
Ahh my apologies I did add a fix to the `block` package that I forgot to add a label for `package: block` in #1581 regarding the initial base fee:

<img width="1238" alt="Screen Shot 2022-02-01 at 11 42 12 AM" src="https://user-images.githubusercontent.com/22116/152039506-3dd5848d-36c3-4807-ac51-7ae592e2a78d.png">

This is related to the EIP-1559 spec that the initial base fee for the London HF block should be `INITIAL_BASE_FEE = 1000000000`. We did run into this in the merge testing actually but this time for `sepolia` I fixed it in the actual block package.

So we should do a patch release for `block`. Just that release might fix the client without needing a client release since `client/package.json` states `@ethereumjs/block` with a carat in the version range, so we can see.
